deploy/charts/rook-ceph-cluster/templates/ingress.yaml

```
@@ -31,6 +31,9 @@ spec:
               {{- else }}
               servicePort: http-dashboard
               {{- end }}
+  {{- if .Values.ingress.dashboard.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.dashboard.host.path | default "nginx" }}
```

No need for the default if this will only be evaluated when ingressClassName is set, right?
```suggestion
  ingressClassName: {{ .Values.ingress.dashboard.host.path }}
```

---

Right, makes sense.

---

Actually, shouldn't the value be set like this?
```suggestion
  ingressClassName: {{ .Values.ingress.dashboard.ingressClassName }}
```

---

forgot to update after copy paste.